rdt-app/src/main/assets/json.form/patient-registration-form.json

```diff
@@ -79,55 +79,163 @@
         }
       },
       {
-        "key": "patient_age_label",
-        "type": "label",
-        "text": "Age",
-        "text_color": "#000000"
+        "key": "sex",
+        "openmrs_entity_parent": "",
+        "openmrs_entity": "person",
+        "openmrs_entity_id": "sex",
+        "type": "native_radio",
+        "label": "Gender",
+        "options": [
+          {
+            "key": "Female",
+            "text": "Female"
+          },
+          {
+            "key": "Male",
+            "text": "Male"
+          }
+        ],
+        "v_required": {
+          "value": "true",
+          "err": "Please specify gender"
+        }
       },
       {
-        "key": "patient_age",
+        "key": "patient_dob",
         "openmrs_entity_parent": "",
         "openmrs_entity": "person_attribute",
+        "openmrs_entity_id": "birthdate",
+        "type": "date_picker",
+        "hint": "Patient's date of birth ",
+        "expanded": false,
+        "duration": {
+          "label": "Age"
+        },
+        "min_date": "today-100y",
+        "max_date": "today",
+        "v_required": {
+          "value": "true",
+          "err": "Please enter the date of birth"
+        }
+      },
+      {
+        "key": "age",
```

Comment:
map all openmrs properties here to empty string so that they are visible in the event obs

Reply:
i.e. remove `person_age` and `age`